modules/telco-core-rds-container.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/core/telco-core-ref-crs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-core-rds-container_{context}"]
+= Extracting the telco core reference design configuration CRs
+
+You can extract the complete set of custom resources (CRs) for the telco core profile from the `telco-core-rds-rhel9` container image. 
+The container image has both the required CRs, and the optional CRs, for the telco core profile.
+
+.Prerequisites
+
+* You have installed `podman`.
+
+.Procedure
+
+* Extract the content from the `telco-core-rds-rhel9` container image by running the following commands:
++
+[source,terminal]
+----
+$ mkdir -p ./out
+----
++
+[source,terminal]
+----
+$ podman run -it registry.redhat.io/openshift4/telco-core-rds-rhel9:v4.16 | base64 -d | tar xv -C out
+----
+
+.Verification
+
+* The `out` directory has the following folder structure. You can view the telco core CRs in the `out/telco-core-rds/` directory.
++
+.Example output
+[source,text]
+----
+out/
+├── argocd
+│   ├── deployment
+│   └── example
+│       ├── acmpolicygenerator
+│       │   └── hub-side-templating
+│       │       └── configMaps
+│       ├── policygentemplates
+│       │   └── hub-side-templating
+│       │       └── configMaps
+│       └── siteconfig
+├── extra-manifest
+│   └── workload
+├── source-crs
+│   ├── extra-manifest
+│   │   └── workload
+│   ├── optional-extra-manifest
+│   └── validatorCRs
+└── telco-core-rds
+    ├── cpu_partitioning
+    ├── logging
+    ├── networking
+    │   ├── metallb
+    │   ├── multinetworkpolicy
+    │   ├── multus
+    │   │   └── tap_cni
+    │   └── sriov
+    ├── other
+    ├── scheduling
+    ├── storage
+    │   └── odf-external
+    └── tuning
```

Comment:
I actually got a different structure when I ran the command locally. perhaps this worth a change since the last commit on the repo separating optional vs required. ALso I think we can point out that we are using tree to display the subfolders/files  under out